raiden/tests/utils/factories.py

```diff
@@ -165,6 +165,10 @@ def make_20bytes() -> bytes:
     return bytes("".join(random.choice(string.printable) for _ in range(20)), encoding="utf-8")
 
 
+def make_locksroot() -> Locksroot:
+    return Locksroot(make_32bytes())
```

Comment:
If we used real python classes instead of `NewType`s, we could just have a proper constructor and call `LocksRoot()` instead of `make_locksroot`. But I'm not sure about the performance impact and such a change would be way out of scope for this PR.

Reply:
True, we should create an issue for discussion or put down that topics to the list of topics that will be discusses at the offsite.